staging/src/k8s.io/apimachinery/pkg/api/operation/operation.go

```diff
@@ -42,6 +42,15 @@ type Operation struct {
 	//
 	// Unset options are disabled/false.
 	Options sets.Set[string]
+
+	// Request provides information about the request being validated.
+	Request Request
+}
+
+// Request provides information about the request being validated.
+type Request struct {
+	// Subresources is the path to the subresource being validated.
```

Comment:
Maybe a remark on how validators should use this?

Reply:
Added